### PR TITLE
fix/page-bugs-correction

### LINKED ISSUE #130: Since activating service worker, page always scroll down to "service worker gateway" section
Since I activated the service worker via js.ipfs.io, whenever I visit js.ipfs.io it jumps down to that section.

I could understand it the first time, as it makes sure the little box on the right is visible after activating the gateway, but on the next loads, I don't think it should scroll down there directly.

Using Arch Linux + Firefox 61.0.1 (64-bit)

### LINKED ISSUE #131: Service worker success dialog alignment issues
2d4115ac-5898-4eb1-963e-f08ab15d2871

The cross should be always placed in the right.

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
- [x] `Community` section description width is small on mobile devices

- [x] Toggle button is not showing the right message when browser does not support service workers. 

- [x] Arrows are faulty as they have one blank px line over them

- [x] `Run` buttons on `Getting started` section are missing border-radius and transition on hover to be consistent

- [x] Alpha banner line height must be increased

- [x] Modal alignments (closes #131)

- [x] Since activating service worker, the page always scroll down to "service worker gateway" section (closes #130) 

- [x] Add `Disable Service worker` text to toggle button when it is activated.

### CODE REVIEW

File: src/shared/components/gateway-section/toggle-button/index.js [hunk 1]
```diff
@@ -21,14 +21,15 @@ class ToggleButton extends Component {
...
+    const toggleText = this.getToggleText()
 
     return (
       <div data-tip data-for='incompatible-sw' className={ wrapperClasses }>
         <label className={ styles.switch } onClick={ this.handleToggleButton }>
           <span className={ sliderClasses } />
         </label>
         <span className={ titleClasses }>
-          { inProgress && !isActive ? messages.serviceWorker.activatingToggleText : messages.serviceWorker.toggleText }
+          { toggleText }
```
Comment (vasco-santos) [inline src/shared/components/gateway-section/toggle-button/index.js]: Why not using the function directly in here?

File: src/shared/components/gateway-section/toggle-button/index.js [hunk 2]
```diff
@@ -44,6 +45,19 @@ class ToggleButton extends Component {
...
   }
+
+  getToggleText = () => {
+    const { isActive, inProgress, intl: { messages } } = this.props
+
+    if (!isActive) {
+      if (!inProgress) {
+        return messages.serviceWorker.toggleText
+      } else {
+        return messages.serviceWorker.activatingToggleText
```
Comment (vasco-santos) [inline src/shared/components/gateway-section/toggle-button/index.js]: You can simplify the the above returns by a single one with a ternary condition.

File: src/shared/components/getting-started-section/index.module.css [hunk 3]
```diff
@@ -131,12 +131,14 @@
...
   outline: none;
   background: rgba(255, 255, 255, 0.2);
+  border-radius: 2px;
   color: #fff;
   font-size: 12px;
   text-align: center;
   line-height: 22px;
   letter-spacing: 0;
   cursor: pointer;
+  transition: all var(--transition-default-duration) ease;
```
Comment (vasco-santos) [inline src/shared/components/getting-started-section/index.module.css]: It is not a good practice to define `all` properties in a transition. Please specify the ones you want to use.

File: intl/messages/en.json [hunk 4]
```diff
@@ -86,7 +86,9 @@
     "activatingToggleText": "Activating Service Worker...",
     "activationErrorMessage": "An error occurred when enabling the Service Worker.",
     "deactivationErrorMessage": "An error occurred when disabling the Service Worker.",
-    "nonRootScopeWarningMessage": "The Service Worker can't be activated when viewing from this URL. Try in https://js.ipfs.io instead."
+    "nonRootScopeWarningMessage": "The Service Worker can't be activated when viewing from this URL. Try in https://js.ipfs.io instead.",
```
Comment (PedroMiguelSS) [inline intl/messages/en.json]: Note to self: add the messages to all languages

**Pull request merged**